tiup/tiup-cluster.md

```diff
@@ -128,7 +144,7 @@ prometheus  172.16.5.134  9090         deploy/prometheus-9090,data/prometheus-90
 grafana     172.16.5.134  3000         deploy/grafana-3000
 Attention:
     1. If the topology is not what you expected, check your yaml file.
-    1. Please confirm there is no port/directory conflicts in same host.
+    2. Please confirm there is no port/directory conflicts in same host.
```

Comment:
This should be fixed in the code too. @lucklove Please be aware of it.

Reply:
I have checked the code, it has been fixed, thank you, keke